roles/sql-command/tasks/create-user.yml

```diff
@@ -68,4 +68,4 @@
     - name: Remove my.cnf file
       include: remove-my-cnf.yml
 
-  when: "{{ rds_dbs | selectattr('db_engine', 'match', 'MySql')| list |count > 0 }}"
+  when: rds_dbs | selectattr('db_engine', 'match', 'MySql')| list |count > 0
```

Comment:
This can be ignored for now @Azulinho, it's being worked through with my current story for the wordpress containers